src/test/CMakeLists.txt

```diff
@@ -619,11 +619,6 @@ if(PROMTOOL_EXECUTABLE)
     message(WARNING "'${PROMTOOL_EXECUTABLE} test rules' does not work, "
       "please use a newer prometheus")
   endif()
-else()
```

Comment:
might want to print a warning noting that `run-promtool-unittests` is skipped due to missing `promtool`.

Reply:
Thanks @tchaikov . Done